vlttng/venv.py

```diff
@@ -547,14 +547,18 @@ def _build_lttng_tools(self):
         if project is None:
             return
 
-        if '--with-lttng-ust-prefix' in project.configure or '--without-lttng-ust' in project.configure:
-            msg = 'I would not pass the --with-lttng-ust-prefix/--without-lttng-ust configure option if I were you: they are handled by vlttng'
-            _pwarn(msg)
+        for ust_config in ['--with-lttng-ust-prefix', '--without-lttng-ust', '--disable-lttng-ust']:
+            if ust_config in project.configure:
+                msg = 'I would not pass the '+ust_config+' configure option if I were you: they are handled by vlttng'
```

Comment:
using `join()` the entire list will be printed for each `lttng_ust_ops` present in the configure options.
If I have both `--with-lttng-ust-prefix`  and `--enable-lttng-ust=yes`, the same warning will be printed twice.
If that's what we want, we should break after the first warning.